### PR TITLE
RFC - Proposed schedule

### PR DESCRIPTION
**Pull request by BKStephens**
How does this schedule work for everyone? Any suggested changes?

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -12,8 +12,16 @@ week | chapters   | pages | comments
 -----|------------|-------|---------
 04/11/16  | Elixir Hack Night | |
 04/25/16  | Elixir Hack Night | |
-05/09/16  | Intro and 1 | ? |
-05/16/16  | Chapter 2 | |
+05/09/16  | Chapter 1 | 12 |
+05/16/16  | Chapter 2 | 21 |
+05/23/16  | Chapter 3 & 4 | 29 |
+05/30/16  | Chapter 5 | 20 |
```
Comment (BKStephens) [inline README.md]: Whoops, forgot this is Memorial Day. Should we not meet on 5/30?

Comment (dazuma) [inline README.md]: Oh yeah, I missed that too. Yeah we should skip Memorial Day.

Comment (marcisme) [inline README.md]: I'm going to be out of town 5/30, 6/6 and the week of 6/13.

I will be here from 6/7 - 6/11, so if we could meet any of those days, I'd be able to attend that week.

File: README.md [hunk 2]
```diff
@@ -12,8 +12,16 @@ week | chapters   | pages | comments
...
+06/06/16  | Chapter 6 & 7 | 34 |
+06/13/16  | Chapter 8 & 9 | 38 |
```
Comment (marcisme) [inline README.md]: This is WWDC week.

**Comment by bpina** [comment]
👍

**Comment by kylerippey** [comment]
👍 

I'll be missing 5/23, but I _should_ be good for all the others.

**Pull request merged**